examples/bri-3/src/bri/auth/services/auth.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+import { AuthAccesorService } from '../agents/authAccessor.agent';
+import { errorMessage } from '../constants';
+import { Login } from '../model/login';
+import { LoginService } from './login.service';
+
+@Injectable()
+export class AuthService {
+  constructor(
+    private readonly authAccesorService: AuthAccesorService,
+    private jwtService: JwtService,
+    private readonly loginService: LoginService,
+  ) {}
+
+  async validateUser(username: string, pass: string): Promise<any> {
+    const user = await this.authAccesorService.findOne(username);
```

Comment:
use bpiSubject terminology instead of user, use publicKey instead of username

why do we need password at all? or verifyUser method?

Reply:
Removed